rskj-core/src/main/java/org/ethereum/rpc/validation/BnTagOrNumberValidator.java

```diff
@@ -0,0 +1,29 @@
+package org.ethereum.rpc.validation;
+
+import co.rsk.util.HexUtils;
+import org.ethereum.core.genesis.BlockTag;
+import org.ethereum.rpc.exception.RskJsonRpcRequestException;
+
+public class BnTagOrNumberValidator {
+    private BnTagOrNumberValidator() {
+
+    }
+
+    public static boolean isValid(String parameter) {
```

Comment:
(just a suggestion) As we are not using return value false, can we make the function just void? Or return false, and handle the exception outside?

Reply:
yes, this is actually what sonar is complaining about

Reply:
just wanted to do it that way for test to have a positive result but it could be switched to void